test/factorial_test.js

```diff
@@ -1,5 +1,5 @@
 import { expect } from 'chai'
-import factorial from '../src/factorial'
+import { factorial, recursiveFactorial } from '../src/factorial'
```

Comment:
We should also test negative cases like `factorial( 0 )` and `factorial( -1 )` to ensure this function works when called with invalid parameters.